src/System Application/App/Extension Management/src/ExtensionInstallationImpl.Codeunit.al

```diff
@@ -364,5 +364,44 @@ codeunit 2500 "Extension Installation Impl"
         ExtensionDetails.Run();
         exit(ExtensionDetails.Editable());
     end;
+
+    internal procedure AllowsDebug(ResourceProtectionPolicy: Integer): Boolean
+    begin
+        exit(HasPolicy(ResourceProtectionPolicy, 1));
+    end;
+
+    internal procedure AllowsDownloadSource(ResourceProtectionPolicy: Integer): Boolean
+    begin
+        exit(HasPolicy(ResourceProtectionPolicy, 2));
+    end;
+
+    internal procedure AllowsDownloadSourceInSymbols(ResourceProtectionPolicy: Integer): Boolean
+    begin
+        exit(HasPolicy(ResourceProtectionPolicy, 4));
+    end;
```

Comment:
The codeunit already is internal, and the other procedures are also defined as global.

```suggestion
    procedure AllowsDebug(ResourceProtectionPolicy: Integer): Boolean
    begin
        exit(HasPolicy(ResourceProtectionPolicy, 1));
    end;

    procedure AllowsDownloadSource(ResourceProtectionPolicy: Integer): Boolean
    begin
        exit(HasPolicy(ResourceProtectionPolicy, 2));
    end;

    procedure AllowsDownloadSourceInSymbols(ResourceProtectionPolicy: Integer): Boolean
    begin
        exit(HasPolicy(ResourceProtectionPolicy, 4));
    end;
```

Reply:
I like being explicit about accessibility but fair point.